### PR TITLE
Add support for chrome.proxy.settings

### PR DESCRIPTION
**Pull request by chanand**
Add support for [`chrome.proxy.settings`](https://developer.chrome.com/extensions/proxy#property-settings) as mentioned in issue #11.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "mocha-lcov-reporter": "^1.0.0",
     "mocha-tldr-reporter": "^1.0.0",
     "sinon": "^1.17.2",
-    "sinon-chrome": "^1.1.0",
+    "sinon-chrome": "^2.1.2",
```
Comment (acvetkov) [inline package.json:51]: @chanand Hi. Can you revert this change?
I'll update this dependency in separate release

Comment (chanand) [reply package.json:51]: @acvetkov, the tests fail with `sinon-chrome v1.1.0`, do you still want me to revert this change??

Comment (acvetkov) [reply package.json:51]: No. Sorry for waiting.

**Comment by acvetkov** [comment]
@chanand  I've published new version 1.0.6
https://www.npmjs.com/package/then-chrome

Thanks for pr

**Pull request merged**